addons/l10n_id_efaktur/models/efaktur.py

```diff
@@ -12,47 +12,63 @@ class Efaktur(models.Model):
     _description = "Available E-faktur range"
```

Comment:
We should add a _rec_names_search here so that searching in the m2o works better.
Something like:

```suggestion
    _description = "Available E-faktur range"
    _rec_names_search = ["min", "max"]
```
Will allow to find a range by typing the min or max number. this should be good enough